src/main/java/gregtech/common/ConfigHolder.java

```diff
@@ -136,8 +136,9 @@ public static class RecipeOptions {
         @Config.Comment({"Whether to make Glass related recipes harder. Default: true"})
         public boolean hardGlassRecipes = true;
 
-        @Config.Comment({"Whether to nerf Wood crafting to 2 Planks from 1 Log, and 2 Sticks from 2 Planks.", "Default: false"})
-        public boolean nerfWoodCrafting = false;
+        @Config.Comment({"Whether to nerf/Buff Wood crafting to X Planks from 1 Log, and X Sticks from 2 Planks.", "Default: 4"})
+        @Config.RangeInt(min = 0, max = 42)
```

Comment:
I don't think this should ever be allowed to go higher than the vanilla value. The nerf is meant to push players towards machines, where there are better rates for their materials. Making 42 planks from 1 log means that players will never really need to make these machines, and also making more planks than the recipe normally gives in vanilla does not really feel right.